sdk/eventhubs/azure-messaging-eventhubs/test/ut/consumer_client_test.cpp

```diff
@@ -385,12 +385,9 @@ namespace Azure { namespace Messaging { namespace EventHubs { namespace Test {
       EXPECT_EQ(totalReceived, numberOfEvents);
 
       // We have consumed all the events. Attempting to consume one more should block.
-      {
-        Azure::Core::Context timeout = Azure::Core::Context::ApplicationContext.WithDeadline(
-            Azure::DateTime::clock::now() + std::chrono::seconds(3));
-        EXPECT_THROW(
-            partitionClient.ReceiveEvents(50, timeout), Azure::Core::OperationCancelledException);
-      }
+      Azure::Core::Context timeout{Azure::DateTime::clock::now() + std::chrono::seconds()};
```

Comment:
Add 3 seconds to keep the behavior identical? Unless this was intentional.
```suggestion
      Azure::Core::Context timeout{Azure::DateTime::clock::now() + std::chrono::seconds(3)};
```

Reply:
It was not intentional, this was accidentally lost.